pegasus/helpers/section_api_helpers.rb

```diff
@@ -105,12 +105,12 @@ def self.fetch_if_allowed_array(ids, dashboard_user_id)
   end
 
   def self.update_if_allowed(params, dashboard_user_id)
-    user_to_update = Dashboard.db[:users].
-      where(id: params[:id], deleted_at: nil)
+    user_to_update = Dashboard.db[:users].where(id: params[:id], deleted_at: nil)
     return if user_to_update.empty?
-    return if Dashboard.db[:followers].
-      where(student_user_id: params[:id], user_id: dashboard_user_id).
-      where(deleted_at: nil).
+    return if Dashboard.db[:sections].
+      join(:followers, section_id: :sections__id).
+      where(followers__student_user_id: params[:id], sections__user_id: dashboard_user_id).
+      where(followers__deleted_at: nil).
```

Comment:
nit: any reason why the first 2 conditions are in one `where`, and the 3rd condition is in a 2nd `where`? I'd prefer consistency, either all comma-separated args in a single `where` or all separate.